base_link: 'base_link'

gripper_group:
  end_effector_name: 'gripper_group'
  joints : ['gripper_finger_joint']
  pregrasp_posture : [0.0]
  grasp_posture : [1.0]
  end_effector_parent_link : 'gripper_roll_link'
  pregrasp_time_from_start : 4.0
  grasp_time_from_start : 4.0
  grasp_pose_to_eef : [0,0,0]
